packages/inquirer/lib/prompts/rawlist.js

```diff
@@ -146,6 +150,35 @@ class RawListPrompt extends Base {
 
     this.render();
   }
+
+  /**
+   * When user press up key
+   */
+
+  onUpKey() {
+    if (this.selected) {
+      this.selected--;
```

Comment:
We'll need to also account for the selected index going under or over the list length.